### ISSUE TITLE
[Feature Request] Export/Import Shazam Fields

### ISSUE THREAD

**Issue by tertek** [susom/redcap-em-shazam#50]
Would be nice to have a button in the "Shazam Fields" config page to export/import field data. Especially usefull when moving from one environment to another.

@susom

**Comment by 123andy** [comment]
I agree 🙂. I never got around to it.  Ugh -- so much to improve on.  Ideally the Shazam config could be stored in the metadata as a massive encoded string (or maybe an external ref)...
________________________________
From: Ekin Tertemiz ***@***.***>
Sent: Thursday, December 1, 2022 12:23 AM
To: susom/redcap-em-shazam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [susom/redcap-em-shazam] [Feature Request] Export/Import Shazam Fields (Issue #50)


Would be nice to have a button in the "Shazam Fields" config page to export/import field data. Especially usefull when moving from one environment to another.

@susom<https://github.com/susom>
